picard/acoustid/__init__.py

```diff
@@ -82,6 +82,10 @@ def init(self):
     def done(self):
         pass
 
+    def get_max_processes(self):
+        config = get_config()
+        return config.setting['fpcalc_threads'] or 2
```

Comment:
We should have this `2` as a constant: `DEFAULT_FPCALC_THREADS = 2` and use it also for default config